cpp/daal/src/externals/service_math_mkl.h

```diff
@@ -135,27 +106,27 @@ struct MklMath<double, cpu>
         return r;
     }
 
-    static void vPowx(SizeType n, const double * in, double in1, double * out) { VMLFN_CALL(dPowx, HAynn, ((int)n, in, in1, out)); }
+    static void vPowx(SizeType n, const double * in, double in1, double * out) { VMLFN_CALL(dPowx, ((int)n, in, in1, out)); }
```

Comment:
MKL math functions support various accuracy modes.
Previously the `mode` parameter was provided as an argument to a function (`HAynn`) and now it is not. This might lead to unexpected changes in the functions results.
Let's provide the mode to the functions explicitly like it was done previously.

To do this you need to call `vmdPowx` instead `vdPowx`, for example.
and provide following set of flags as the additional argument at the end of the arguments list: `VML_HA | VML_FTZDAZ_CURRENT | VML_ERRMODE_DEFAULT`.
See https://www.intel.com/content/www/us/en/docs/onemkl/developer-reference-c/2023-2/v-powx.html